packages/graphql-auth-transformer/src/ModelAuthTransformer.ts

```diff
@@ -130,6 +130,7 @@ const validateAuthModes = (authConfig: AppSyncAuthConfiguration) => {
 
 export type ModelAuthTransformerConfig = {
   authConfig?: AppSyncAuthConfiguration;
+  isAmplifyAdminApp?: Boolean;
```

Comment:
the gql transformer is theoretically "amplify agnostic" so can we name this param something else? maybe `allowPrivateIAMAuth`?